packages/picasso/src/List/styles.ts

```diff
@@ -1,10 +1,17 @@
-import { createStyles } from '@material-ui/core/styles'
+import { createStyles, Theme } from '@material-ui/core/styles'
 
-export default () =>
+export default ({ palette }: Theme) =>
   createStyles({
     root: {
       listStyle: 'none',
       padding: 0,
-      margin: 0
+      margin: 0,
+
+      '& > li:not(:last-child)': {
+        marginBottom: '0.5em'
+      }
+    },
+    ul: {
+      color: palette.grey.dark
```

Comment:
```suggestion
      color: palette.text
```

It is the same color, but more specific